src/components/Day.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import activityPropTypeShape from '../prop-types/activity';
+import ActivitiesListItem from './ActivitiesListItem';
+
+const Day = ({ activities, weekDay }) => (
+  <div>
+    <p>{weekDay.toUpperCase()}</p>
+    {activities.map(activity => (
+      <ActivitiesListItem {...activity} key={activity.id} />
```

Comment:
if you will pass `activity` itself, it will be better for performance and better for bugs. Why?
- now, you are creating new variables on each rerender, as fields of the activities are primitives. 
- object is passed by reference, so in RAM you will have only one place taken by it